src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -117,13 +140,15 @@ const NavHeader = (props) => {
           )}
         </View>
       </View>
-      <Animated.View style={styles.Header}>
+      <Animated.View style={[styles.headerContainer]}>
         {!isHeaderHide ? (
           <>
-            <Animated.Image
-              source={topicDetail?.icon_path ? {uri: topicDetail?.icon_path} : TopicDefaultIcon}
-              style={styles.image(opacityHeaderAnimation)}
-            />
+            <Animated.View style={{opacity: opacityHeaderAnimation}}>
+              <FastImage
+                source={topicDetail?.icon_path ? {uri: topicDetail?.icon_path} : TopicDefaultIcon}
+                style={styles.image}
+              />
+            </Animated.View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `FastImage` component is used instead of `Animated.Image`, and the `opacity` style is moved to a wrapping `Animated.View`. Ensure that this change doesn't affect the animation or layout.
```diff
-            <Animated.Image
+            <Animated.View style={{opacity: opacityHeaderAnimation}}>
+              <FastImage
                 source={topicDetail?.icon_path ? {uri: topicDetail?.icon_path} : TopicDefaultIcon}
-                style={styles.image(opacityHeaderAnimation)}
+                style={styles.image}
+              />
+            </Animated.View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->